README.md

```diff
@@ -106,7 +106,7 @@ follows:
 
 ```
 curl -LO --output-dir packages https://github.com/dicej/spin-dotnet-sdk/releases/download/canary/aspnetcore-wasi.zip
```

Comment:
```suggestion
curl -LO https://github.com/dicej/spin-dotnet-sdk/releases/download/canary/aspnetcore-wasi.zip
```

I shouldn't have blindly cut-and-paste the `curl` command; I didn't mean to specify the output directory.  How about we remove that option and then leave line 109 unchanged?

Reply:
Sure!